shell/platform/fuchsia/flutter/BUILD.gn

```diff
@@ -14,17 +14,17 @@ import("//flutter/tools/fuchsia/fuchsia_libs.gni")
 import("//flutter/vulkan/config.gni")
 import("engine_flutter_runner.gni")
 
-shell_gpu_configuration("fuchsia_gpu_configuration") {
+# Fuchsia uses its own custom Surface implementation.
+shell_gpu_configuration("fuchsia_legacy_gpu_configuration") {
   enable_software = false
   enable_gl = false
-  enable_vulkan = true
+  enable_vulkan = false
```

Comment:
why is this changing?

Reply:
These flags control the linking-in of the various GpuSurface* targets in shell/gpu.  We don't actually use any of those targets because we use our custom Surface+VulkanSurfaceProducer; and they were interfering with the template() trick I'm doing in this CL.

I'm not sure if this causes a binary size reduction, but this is removing dead code

Reply:
BTW: the flutter_runner_next will use the "embedder_gpu_configuration" in shell/platform/embedder/BUILD.gn  I'll enable the Vulkan support for it there.

This configuration will only be used for the current (legacy) runner.

Reply:
ack. thanks for explaining!